.gitignore

```diff
@@ -1 +1,2 @@
 /Debug/*
+/Debug/
```

Comment:
Remove this line. We'll run into a bunch of problems when pushing/pulling if this line stays there I think

Reply:
To do this, go to the workspace folder, then type in vim .gitignore, then press i, then get rid of the line, then press esc, then type in :wq and hit enter